src/Lykke.Job.TransactionHandler/Projections/OperationHistoryProjection.cs

```diff
@@ -236,7 +235,7 @@ public class OperationHistoryProjection
 
         public async Task Handle(ForwardWithdawalLinkedEvent evt)
         {
-            await _log.WriteInfoAsync(nameof(OperationHistoryProjection), nameof(ForwardWithdawalLinkedEvent), evt.ToJson(), "");
+            _log.WriteInfo(nameof(ForwardWithdawalLinkedEvent), evt.ToJson(), "");
```

Comment:
This is not needed with latest version of Lykke.Cqrs - such logging is done from the library